### ISSUE TITLE
How to write a Las file with the library?

### ISSUE THREAD

**Issue by jabrena** [jsimomaa/LASlibJNI#2]
Hi,

I would like to test the library but I don't know how to write a las file using the library.

Any example?

Juan Antonio

**Comment by jsimomaa** [comment]
@jabrena unfortunately currently writing of LAS files is unsupported. This could be added by extending the JNI

**Comment by jabrena** [comment]
Are you talking about this path?
https://github.com/jsimomaa/LASlibJNI/tree/master/jni

**Comment by jsimomaa** [comment]
@jabrena Yes, `org_lastools_LASWriter.h` and `class_LASWriter.cpp` should be introduced and implemented accordingly. Furthermore, a java class like this should be implemented [LASReader](https://github.com/jsimomaa/LASlibJNI/blob/master/src/main/java/org/lastools/LASReader.java) but for writer
